.github/workflows/nightly_tests.yml

```diff
@@ -99,7 +96,7 @@ jobs:
       if: ${{ always() }}
       uses: actions/upload-artifact@v2
       with:
-        name: ctrd-logs
+        name: ${{ github.job }}-${{ matrix.vhive_args }}-ctrd-logs
```

Comment:
Why use github job here as the log directory uses only `$GITHUB_RUN_ID` as the identifier?